apps/console/src/features/groups/components/group-common-components/render-chip.tsx

```diff
@@ -31,17 +30,21 @@ interface RenderChipInterface extends IdentifiableComponentInterface, ChipProps
      */
     setActiveOption: (option: RolesMemberInterface) => void;
     /**
-     * Primary text of the chip.
+     * Display name of the role
```

Comment:
If we are updating the component properties specifically for displaying role information, let's also rename `RenderChip` and related interface names as well.

Reply:
Fixed with 8cb8bf2176f03d3406c2b5b4530b56bf77c07a78